### PR TITLE
Gstreamer underrun

### PR DESCRIPTION
**Pull request by JeanTracker**
- Gstreamer: handle for underrun
- AudioPlayer: notify to report player's event

### CODE REVIEW

File: src/capability/audio_player_agent.cc [hunk 1]
```diff
@@ -1169,6 +1175,9 @@ void AudioPlayerAgent::mediaEventReport(MediaPlayerEvent event)
         mediaStateChanged(MediaPlayerState::STOPPED);
         capa_helper->releaseFocus("cap_audio");
         break;
+    case MediaPlayerEvent::PLAYING_MEDIA_UNDERRUN:
+    for (auto aplayer_listener : aplayer_listeners)
+        aplayer_listener->mediaEventReport(AudioPlayerEvent::UNDERRUN, cur_dialog_id);
```
Comment (kimhyungrok) [inline src/capability/audio_player_agent.cc:1180]: Is there need "break" keyword?

File: include/capability/audio_player_interface.hh [hunk 2]
```diff
@@ -46,6 +46,16 @@ enum class AudioPlayerState {
     FINISHED /**< Status playback finished in AudioPlayer */
 };
 
+/**
+ * @brief AudioPlayerEvent
+ */
+enum class AudioPlayerEvent {
+    UNDERRUN, /**< This event is occurred when the content is reloaded because of bad quailty network */
+    LOAD_FAILED, /**< This event is occurred when the content is not loading successfully */
```
Comment (kimhyungrok) [inline include/capability/audio_player_interface.hh:54]: Is it correct "is not loaded" ?

File: src/capability/audio_player_agent.cc (hunk 1 above)
Comment (JeanTracker) [reply src/capability/audio_player_agent.cc:1180]: Thanks. I'll fix it.

File: include/capability/audio_player_interface.hh (hunk 2 above)
Comment (JeanTracker) [reply include/capability/audio_player_interface.hh:54]: Thanks. I'll fix it.

File: src/core/media_player.cc [hunk 3]
```diff
@@ -151,6 +151,11 @@ MediaPlayer::MediaPlayer(int volume)
             for (auto l : d->listeners)
                 l->mediaEventReport(MediaPlayerEvent::PLAYING_MEDIA_FINISHED);
             break;
+        case NUGU_MEDIA_EVENT_MEDIA_UNDERRUN:
+            for (auto l : d->listeners)
+                l->mediaEventReport(MediaPlayerEvent::PLAYING_MEDIA_UNDERRUN);
+        break;
```
Comment (webispy) [inline src/core/media_player.cc:157]: Please fix indentation

Comment (JeanTracker) [reply src/core/media_player.cc]: Thanks. I'll fix it.

**Pull request merged**